### PR TITLE
added sample yaml for gitops

### PR DESCRIPTION
**Pull request by jc-berger**
see jira: https://issues.redhat.com/browse/ACM-10457

### CODE REVIEW

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Jake!

File: gitops/gitops_push_pull.adoc [hunk 1]
```diff
@@ -17,14 +17,35 @@ For both models, the same `ApplicationSet` CRD is used to deploy the application
...
+
+* If your `openshift-gitops-ArgoCD-application-controller` service account is _not_ assigned as a cluster administrator, the GitOps application controller might not deploy resources. The application status might send an error similar to the following error:
 
 ----
 cannot create resource "services" in API group "" in the namespace
 "mortgage",deployments.apps is forbidden: User
 "system:serviceaccount:openshift-gitops:openshift-gitops-Argo CD-application-controller"
 ----
 
+* To add cluster administrator privileges to your service account, see the following example yaml:
```
Comment (dockerymick) [inline gitops/gitops_push_pull.adoc]:
```suggestion
* To add cluster administrator privileges to your service account, see the following example YAML: 
```

**Review by dockerymick — APPROVED** [review]
One suggestion, thanks for the updates!

File: gitops/gitops_push_pull.adoc (hunk 1 above)
Comment (netzzer) [reply gitops/gitops_push_pull.adoc]: Also need to make it clear that the YAML file is created on managed clusters that could be selected to create an application using the Pull model.

**Review by netzzer — CHANGES_REQUESTED** [review]

**Pull request merged**